### PR TITLE
netpbm: new package

### PR DESCRIPTION
**Pull request by cessenat**
Netpbm - graphics tools and converters.
This provides all netpbm tools, many of them are used by "LaTeX2HTML".
The perl configure and install scripts are not used since they are interactive and talkative.

### CODE REVIEW

File: var/spack/repos/builtin/packages/netpbm/package.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        if spec.target.family == 'x86_64':
+            config.append('WANT_SSE = Y')
+
+        with open('config.mk', 'a') as mk:
+            mk.write('\n'.join(config))
+
+    def build(self, spec, prefix):
+        make()
+
+    def install(self, spec, prefix):
```
Comment (haampie) [inline var/spack/repos/builtin/packages/netpbm/package.py:174]: There is no make install available?

Comment (cessenat) [reply var/spack/repos/builtin/packages/netpbm/package.py:174]: No, there is a perl script buildtools/installnetpbm.pl that is called from ./installnetpbm
alike for configure.
Another option would be to make these perl scripts batch capable (as opposed to purely interactive and talkative) but I found easier to reimplement the stuff in the spack package.
Is it inconvenient for spack to have so long a package ?

Comment (haampie) [reply var/spack/repos/builtin/packages/netpbm/package.py:174]: The only issue could be that the file structure changes between versions, so you'd have to change the package.py to support multiple versions at the same time, which isn't great. But we can decide on that when they actually change what they install.

Comment (cessenat) [reply var/spack/repos/builtin/packages/netpbm/package.py:174]: Absolutely right, I know it is rather fragile. The good point is that Bryan Henderson the maintainer of netpbm is very reactive.

**Comment by haampie** [comment]
```

var/spack/repos/builtin/packages/netpbm/package.py:7: [I100] Import statements are in the wrong order. 'from shutil import copyfile' should be before 'from spack import *' and in a different group.
var/spack/repos/builtin/packages/netpbm/package.py:7: [I201] Missing newline between import groups. 'from shutil import copyfile' is identified as Stdlib and 'from spack import *' is identified as Application.
var/spack/repos/builtin/packages/netpbm/package.py:8: [I100] Import statements are in the wrong order. 'import os' should be before 'from shutil import copyfile'
```

**Review by haampie — APPROVED** [review]

**Comment by cessenat** [comment]
ci failed, retrying...

**Comment by cessenat** [comment]
Hello @haampie sorry for disturbing again but could you netpbm again where there has been no change at all, only the ci/pipeline seems alive now... Thanks a lot.

**Comment by cessenat** [comment]
Hello @haampie, could you merge netpbm manually ? I would like to add latex2html that depends on it... Thanks for your time.

**Pull request merged**